extensions/grpc/runtime/src/main/java/io/quarkus/grpc/spi/ChannelBuilderCustomizer.java

```diff
@@ -0,0 +1,26 @@
+package io.quarkus.grpc.spi;
+
+import java.util.Map;
+
+import io.grpc.ManagedChannelBuilder;
+import io.quarkus.grpc.runtime.config.GrpcClientConfiguration;
+import io.vertx.grpc.client.GrpcClientOptions;
+
+/**
+ * Allow for customization of Channel building (see GitHub issue #16241 for more details).
+ * Implement the customize method, depending on which Channel implementation you're going to use,
+ * e.g. Vert.x or Netty.
+ * This is an experimental SPI, subject to change.
```

Comment:
It's not really an SPI is it? Isn't it meant to be used by applications as CDI beans?